docs/hw/memory/g80-surface.rst

```diff
@@ -99,16 +99,16 @@ Or, alternatively, the address of byte (x,y) is::
     address + pitch * y + x
 
 
-Tiled surfaces
-==============
+Blocklinear surfaces
+====================
 
-A tiled surface is a 3d array of elements, stored in memory in units called
-"tiles". There are two levels of tiling. The lower-level tile is called
-a "roptile" and has a fixed size. This size is 64 bytes × 4 × 1 on G80:GF100
-cards, 64 bytes × 8 × 1 for GF100+ cards. The higher-level tile is called
-a bigtile, and is of variable size between 1×1×1 and 32×32×32 roptiles.
+A blocklinear surface is a 3d array of elements, stored in memory in units
+called "tiles" or "blocks". There are two levels of tiling. The lower-level
+tile is called a "roptile" and has a fixed size. This size is 64 bytes × 4 × 1
+on G80:GF100 cards, 64 bytes × 8 × 1 for GF100+ cards. The higher-level tile is
+called a bigtile, and is of variable size between 1×1×1 and 32×32×32 roptiles.
```

Comment:
As far as I can tell, nvidia names are thus: lower-level tile -> gob, higher-level tile -> block. All references to "tile" can be removed, nv doesn't use that terminology for G80+.